src/main/kotlin/gg/essential/vigilance/gui/settings/ColorPicker.kt

```diff
@@ -265,24 +265,30 @@ class ColorPicker(initial: Color, allowAlpha: Boolean) : UIContainer() {
             first = false
         }
 
-        graphics.drawDirect()
+        graphics.build()?.drawAndClose(pipeline)
         cleanupDraw()
     }
 
+    private val pipeline = URenderPipeline.builderWithDefaultShader("vigilance:color_picker", UGraphics.DrawMode.QUADS, UGraphics.CommonVertexFormats.POSITION_COLOR).apply {
+        blendState = BlendState.ALPHA.copy(dstAlpha = BlendState.Param.ZERO)
```

Comment:
FYI the blend function used by the original code is just plain wrong. Setting dstAlpha to 0 doesn't make sense.